website/docs/Queries.md

```diff
@@ -355,6 +359,57 @@ render(<Component />);
 const element = screen.getByA11yValue({ min: 40 });
 ```
 
+## Hidden accessibility option
+
+All queries have the `hidden` option which enables them to respect accessibility props on components when it is set to `false`. If you set `hidden` to `true`, elements that are normally excluded from the accessibility tree are considered for the query as well.  For now if no `hidden` option is defined, `hidden` is set to `true` by default in order to avoid breaking changes. In the near future, we plan to set it to `false` by default so that the default behavior of all queries is to respect accessibility.
+
+An element will be considered inaccessible when it checks one of the following: 
+- it has a style with `display: "none"`
+- it has the prop [`importantForAccessibility='no-hide-descendants'`](https://reactnative.dev/docs/accessibility#importantforaccessibility-android)
+- it has the prop [`accessibilityElementsHidden={true}`](https://reactnative.dev/docs/accessibility#accessibilityelementshidden-ios) 
+- one of its siblings has the prop [`accessibilityViewIsModal={true}`](https://reactnative.dev/docs/accessibility#accessibilityviewismodal-ios)
+- one of its parents is inaccessible due to one of the checks above
```

Comment:
```suggestion
An element are considered to be hidden from accessibility based on [`isInaccessible()`](./API.md#isinaccessible) function result.
```

Reply:
oh indeed i did not see we had docs for isInaccessible